Subject: Don't try to use Sphinx extensions we don't have

sphinx_autosummary_accessors is not packaged;
jupyter_sphinx is but probably doesn't work.

Author: Andreas Tille <tille@debian.org>, Rebecca N. Palmer
Last-Update: Fri, 13 Jan 2023 11:20:28 +0100
---
 docs/requirements-docs.txt         | 10 ----------
 docs/source/10-minutes-to-dask.rst |  8 ++++----
 docs/source/conf.py                |  8 ++++----
 docs/source/scheduling.rst         |  2 +-
 4 files changed, 9 insertions(+), 19 deletions(-)

--- dask.orig/docs/requirements-docs.txt
+++ dask/docs/requirements-docs.txt
@@ -4,22 +4,11 @@
 sphinx-click
 sphinx-copybutton
 sphinx-remove-toctrees
-sphinx_autosummary_accessors
 sphinx-tabs
 sphinx-design
-jupyter_sphinx
-# FIXME: `sphinxcontrib-*` pins are a workaround until we have sphinx>=5.
-#        See https://github.com/dask/dask-sphinx-theme/issues/68.
-sphinxcontrib-applehelp>=1.0.0,<1.0.7
-sphinxcontrib-devhelp>=1.0.0,<1.0.6
-sphinxcontrib-htmlhelp>=2.0.0,<2.0.5
-sphinxcontrib-serializinghtml>=1.1.0,<1.1.10
-sphinxcontrib-qthelp>=1.0.0,<1.0.7
 toolz
 cloudpickle>=1.5.0
 pandas>=1.4.0
-git+https://github.com/dask/distributed
-git+https://github.com/dask-contrib/dask-expr
 fsspec
 scipy
 pytest
--- dask.orig/docs/source/10-minutes-to-dask.rst
+++ dask/docs/source/10-minutes-to-dask.rst
@@ -91,7 +91,7 @@
 
       See :doc:`array`.
 
-      .. jupyter-execute::
+      .. code-block:: python
 
          import numpy as np
          import dask.array as da
@@ -105,12 +105,12 @@
 
       Here are some key properties of a Dask Array:
 
-      .. jupyter-execute::
+      .. code-block:: python
 
          # inspect the chunks
          a.chunks
 
-      .. jupyter-execute::        
+      .. code-block:: python
             
          # access a particular block of data
          a.blocks[1, 3]
@@ -164,7 +164,7 @@
    .. tab-item:: Array
       :sync: array
 
-      .. jupyter-execute::
+      .. code-block:: python
 
          a[:50, 200]
 
--- dask.orig/docs/source/conf.py
+++ dask/docs/source/conf.py
@@ -22,7 +22,7 @@
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
 import sys
 
-import sphinx_autosummary_accessors
+# import sphinx_autosummary_accessors
 
 # -- General configuration -----------------------------------------------------
 
@@ -41,7 +41,7 @@
     "sphinx.ext.mathjax",
     "sphinx.ext.intersphinx",
     "sphinx.ext.autosummary",
-    "sphinx_autosummary_accessors",
+#    "sphinx_autosummary_accessors",
     "sphinx.ext.extlinks",
     "sphinx.ext.viewcode",
     "numpydoc",
@@ -51,7 +51,7 @@
     "sphinx_remove_toctrees",
     "IPython.sphinxext.ipython_console_highlighting",
     "IPython.sphinxext.ipython_directive",
-    "jupyter_sphinx",
+#    "jupyter_sphinx",
     "sphinx_copybutton",
     "sphinx_design",
 ]
@@ -70,7 +70,7 @@
 mathjax_path = 'file:///usr/share/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML'
 
 # Add any paths that contain templates here, relative to this directory.
-templates_path = ["_templates", sphinx_autosummary_accessors.templates_path]
+#templates_path = ["_templates", sphinx_autosummary_accessors.templates_path]
 
 # The suffix of source filenames.
 source_suffix = ".rst"
--- dask.orig/docs/source/scheduling.rst
+++ dask/docs/source/scheduling.rst
@@ -244,7 +244,7 @@
 This import initialization can lead to subprocesses recursively creating other subprocesses
 and eventually an error is raised.
 
-.. dropdown:: Common error encountered
+Common error encountered
 
    .. code-block:: python
 
